sdk/python/feast/entity.py

```diff
@@ -210,7 +229,6 @@ def to_proto(self) -> EntityProto:
 
         spec = EntitySpecProto(
             name=self.name,
-            value_type=self.value_type.value,
```

Comment:
it looks like we're removin value type completely and not deprecating it. should we write it to registry for a while?

Reply:
good catch